portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,33 @@
+package com.google.sps.data;
+
+import java.util.Date;
+
+public class Comment {
```

Comment:
This looks great. One suggestion for extra credit would be to implement a CommentBuilder class (Builder design patttern). Which would allow you to call CommentBuilder.setId(id).setName(name).setDescription(description).setPostDate(date).build()

Reply:
The reason for this is these long list of args to a constructor can get confused

Reply:
Wow! I did that and the I like the initiation process of comments much more now! :)